andy/src/main/java/nl/tudelft/cse1110/andy/execution/metatest/library/evaluators/StringReplacementEvaluator.java

```diff
@@ -12,8 +12,8 @@ public StringReplacementEvaluator(String old, String replacement) {
 
     @Override
     public String evaluate(String oldLibraryCode) {
-        String shiftedOld = this.old.replaceAll("(?m)^\\s+", "");
-        String shiftedOldLibraryCode = oldLibraryCode.replaceAll("(?m)^\\s+", "");
+        String shiftedOld = this.old.replaceAll("(?m)^\\s+|(?m) +$", "");
+        String shiftedOldLibraryCode = oldLibraryCode.replaceAll("(?m)^\\s+|(?m) +$", "");
```

Comment:
Perhaps it would be better to use `\s+` instead of a literal space since that would also cover tabs.

Reply:
Thank you! Fixed it